docset/winserver2022-ps/activedirectory/Get-ADComputer.md

```diff
@@ -163,47 +193,64 @@ USER01-SRV5                                               3/12/2009 7:05:45 PM
 This command gets all the computers that have changed their password in the last 90 days.
 
 ### Example 4: Get computer accounts in a specific location using an LDAPFilter
+
 ```powershell
-PS C:\> Get-ADComputer -LDAPFilter "(name=*laptop*)" -SearchBase "CN=Computers,DC= User01,DC=com"
+Get-ADComputer -LDAPFilter "(name=*laptop*)" -SearchBase "CN=Computers,DC= User01,DC=com"
+```
+
+```Output
 name
 ----
 pattiful-laptop
 davidche-laptop
 ```
 
-This command gets the computer accounts in the location CN=Computers,DC=User01,DC=com that are listed as laptops by using an *LDAPFilter*.
+This command gets the computer accounts in the location
+CN=Computers,DC=User01,DC=com that are listed as laptops by using an
+_LDAPFilter_.
```

Comment:
Here we want to wrap the value in backticks because it's an exact value used in the prompt.

```suggestion
`CN=Computers,DC=User01,DC=com` that are listed as laptops by using an
**LDAPFilter**.
```